### ISSUE TITLE
This XML file does not appear to have any style information associated with it. The document tree is shown below.

### ISSUE THREAD

**Issue by stevena9475** [ks888/LambStatus#141]
I have installed as per the instructions but when i click the link in the email i get the above error page.

I have done it twice now and im sure im not missing any steps, please can you help me?

**Comment by ks888** [comment]
Does the page show an AccessDenied error? If so, please click the link again in 10-20 minutes.

**Comment by nk2580** [comment]
i have this problem but it looks to be an issue with the CloudFront user not being the owner of the content in the S# bucket, i've modified the stack instead because otherwise it just wont work

**Comment by exoqrtx** [comment]
I think I'm having the same issue.  I followed the directions in the setup, received the email from, but when I click on the link i get the same issue as above with error code "AccessDenied".  CloudFormation is showing that everything is complete though. @nk2580 what did you change exactly to get this to work?

**Comment by nk2580** [comment]
the fix i posted above didn't end up working, basically the cloud front user doesn't have the correct permissions.

**Comment by exoqrtx** [comment]
ok, that was my initial thought as well but was hoping for a more surgical solution.

**Comment by exoqrtx** [comment]
Yep, I opened up the permissions and turned on web hosting and it's working.  Kind of a nuclear option, but good enough for now.  I'd still like to find a better way though.

**Comment by bogdan-craciun** [comment]
> Yep, I opened up the permissions and turned on web hosting and it's working. Kind of a nuclear option, but good enough for now. I'd still like to find a better way though.

Where did you change permissions? I didn't find any. Thanks

**Comment by danwhitston** [comment]
Yup, I have also hit this issue, while using the easy install setup. I waited an extra hour but the link is quite firmly producing an XML response with AccessDenied code. I'm going to have a bash at finding the S3 buckets and changing permissions, but this could use more info in the troubleshooting section.

**Comment by danwhitston** [comment]
For anyone else trying to resolve this, go to the S3 service in AWS, i.e. https://s3.console.aws.amazon.com/s3/home. Then for each of the two buckets beginning `statuspage`:

1. Enter the bucket
1. Select all contents using the ticky-box
1. Select Actions / Make public
1. Confirm 'make public'

That's all! At least in my case, I didn't need to play around with turning on web hosting.

**Comment by blckrainbow** [comment]
Can someone help me? I am getting this exact message in an xml file whenever I input the security code sent to my phone so that I can pay by debit card online. I haven't been able to make purchases because of this for like a month now and the bank has no idea what's wrong because my card is working. I cleared the cookies and cache from google chrome but still nothing. I have the card linked to paypal + a bill paying app and they both work fine. I appreciate any tips you might have!
